src/Tests/dotnet-workload-install.Tests/GivenNetSdkManagedWorkloadInstall.cs

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using FluentAssertions;
+using Microsoft.DotNet.Cli.NuGetPackageInstaller;
+using Microsoft.DotNet.ToolPackage;
+using Microsoft.DotNet.Workloads.Workload.Install;
+using Microsoft.NET.Sdk.WorkloadManifestReader;
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Utilities;
+using NuGet.Versioning;
+using Xunit;
+using Xunit.Abstractions;
+using static Microsoft.NET.Sdk.WorkloadManifestReader.WorkloadResolver;
+
+namespace Microsoft.DotNet.Cli.Workload.Install.Tests
+{
+    public class GivenNetSdkManagedWorkloadInstall : SdkTest
+    {
+        private readonly BufferedReporter _reporter;
+
+        public GivenNetSdkManagedWorkloadInstall(ITestOutputHelper log) : base(log)
+        {
+            _reporter = new BufferedReporter();
+        }
+
+        [Fact]
+        public void GivenManagedInstallItsInsallationUnitIsPacks()
+        {
+            var (_, installer, _) = GetTestInstaller();
+            installer.GetInstallationUnit().Should().Be(InstallationUnit.Packs);
+        }
+
+        [Fact]
+        public void GivenManagedInstallItCanGetFeatureBands()
```

Comment:
I think `GetFeatureBandsWithInstallationRecords` should be changed so it only returns bands where there are actually workload installation records.  So this test should be updated to account for that, and to test both empty and non-empty `InstalledWorkloads` directories.